src/index.js

```diff
@@ -1,12 +1,32 @@
 import React from 'react'
 import { render } from 'react-dom'
-import { createStore } from 'redux'
+import { createStore, applyMiddleware } from 'redux'
 import { Provider } from 'react-redux'
 import App from './containers/App'
 import reducer from './reducers'
 import 'todomvc-app-css/index.css'
 
-const store = createStore(reducer)
+import { createCaptureMiddleware, createPersistHandler, createRetrieveHandler, createRetrieveMiddleware, createReplayMiddleware, wrapReducer } from 'redux-vcr';
+
+const firebaseAuth = {
+  apiKey: "AIzaSyBklunZBwf0YMFbpHWVmfcu2scFAb2gq3o",
+  authDomain: "todomvc-ccb8e.firebaseapp.com",
+  databaseURL: "https://todomvc-ccb8e.firebaseio.com",
+};
```

Comment:
Unlike many secrets, Firebase credentials are safe to store in source control. This is because they offer no authorization themselves, they exist purely for identification purposes.